src/osd/OSDMap.cc

```diff
@@ -858,52 +858,52 @@ void OSDMap::Incremental::dump(Formatter *f) const
 
   f->open_array_section("new_up_thru");
 
-  for (const auto &elem : new_up_thru) {
+  for (const auto &up_thru : new_up_thru) {
     f->open_object_section("osd");
-    f->dump_int("osd", elem.first);
-    f->dump_int("up_thru", elem.second);
+    f->dump_int("osd", up_thru.first);
+    f->dump_int("up_thru", up_thru.second);
     f->close_section();
   }
   f->close_section();
 
   f->open_array_section("new_lost");
 
-  for (const auto &elem : new_lost) {
+  for (const auto &lost : new_lost) {
     f->open_object_section("osd");
-    f->dump_int("osd", elem.first);
-    f->dump_int("epoch_lost", elem.second);
+    f->dump_int("osd", lost.first);
+    f->dump_int("epoch_lost", lost.second);
     f->close_section();
   }
   f->close_section();
 
   f->open_array_section("new_last_clean_interval");
 
-  for (const auto &elem : new_last_clean_interval) {
+  for (const auto &last_clean_interval : new_last_clean_interval) {
     f->open_object_section("osd");
-    f->dump_int("osd", elem.first);
-    f->dump_int("first", elem.second.first);
-    f->dump_int("last", elem.second.second);
+    f->dump_int("osd", last_clean_interval.first);
+    f->dump_int("first", last_clean_interval.second.first);
+    f->dump_int("last", last_clean_interval.second.second);
     f->close_section();
   }
   f->close_section();
 
   f->open_array_section("new_blacklist");
-  for (const auto &elem : new_blacklist) {
+  for (const auto &blacklist : new_blacklist) {
```

Comment:
The new_blacklist is declared as:
`    map<entity_addr_t,utime_t> new_blacklist;
`
If so, `blacklist.first` would be the address and `blacklist.second` would be the expire times. This is similar for `old_blacklist` too. I should have discuss this before I made the previous modifications to the *blacklist templates. Please share your thoughts and clarify. Thanks.